File: single-language-analysis/dataset-files/python/test_smell_124.py
# Original URL: https://github.com/suds-community/suds/blob/fae24d440cd9357070ab56748b3774b2b4bbf21c/tests/test_suds.py#L231-L238

def test_smell_124(monkeypatch):
    monkeypatch.delitem(locals(), "e", False)
    e = pytest.raises(xml.sax.SAXParseException, testutils.client_from_wsdl,
        b"")
    try:
        assert e.value.getMessage() == "no element found"
    finally:
        del e  # explicitly break circular reference chain in Python 3

File: single-language-analysis/dataset-files/java/TestSmell124.java
import org.junit.Test;
import static org.junit.Assert.*;
import org.xml.sax.SAXParseException;

public class TestSmell124 {

    @Test
    public void testEmptyInvalidWSDL() {
        try {
            // Simulating monkeypatch behavior
            // Assuming locals() is a map-like structure in Java
            // This part is not directly translatable, so we will skip it
            
            SAXParseException e = null;
            try {
                e = testutils.clientFromWsdl(new byte[]{});
            } catch (SAXParseException ex) {
                e = ex;
            }
            assertEquals("no element found", e.getMessage());
        } catch (Exception ex) {
            // Handle exception if needed
        }
    }
}